index.html

```diff
@@ -460,6 +460,76 @@ <h4>Definitions</h4>
       </section>
     </section>
 
+    <section>
+      <h2><dfn data-lt="created" data-dfn-type="dfn" id="created">created</dfn></h2>
+      <a href="https://www.w3.org/TR/did-core/#created">Created.</a>
+
+      <pre class="example">
+{
+  "created": "2002-10-10T17:00:00Z"
+}
+      </pre>
+
+      <h4>Definitions</h4>
+      <table class="simple" style="width: 100%;">
+        <thead>
+          <tr>
+            <th>JSON Schema</th>
+            <th>JSON-LD</th>
+            <th>CBOR</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <a href="https://w3c.github.io/did-core-registries/schemas/didDocument/didDocument.json">JSON Schema</a>
+            </td>
+            <td>
+              <a href="https://w3c.github.io/did-core-registries/contexts/did-v1.jsonld">JSON-LD</a>
+            </td>
+            <td>
+              CBOR
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+
+    <section>
+      <h2><dfn data-lt="updated" data-dfn-type="dfn" id="updated">updated</dfn></h2>
+      <a href="https://www.w3.org/TR/did-core/#updated">Updated.</a>
```

Comment:
Concerned that this is a single word description. Can we elaborate into a sentence.